tests/Controller/EnMarche/AdherentControllerTest.php

```diff
@@ -23,6 +23,7 @@
 
 /**
  * @group functional
+ * @group controller-1
```

Comment:
You can use meaningful name like `security`, `miscellaneous` etc.

Reply:
you are right, but i just want to create 3 groups (or 2 tags), and these few controller class all have different meanings. and i dont want my command to be very long because i have to manage tags

Reply:
+1